manticore/platforms/linux.py

```diff
@@ -1730,6 +1733,8 @@ def sys_mmap(self, address, size, prot, flags, fd, offset):
             # FIXME Check if file should be symbolic input and do as with fd0
             result = cpu.memory.mmapFile(address, size, perms, self.files[fd].name, offset)
 
+        cpu._publish('did_map_memory', address, size, perms, "" if fd == 0 or flags & 0x20 else self.files[fd].name, offset, result)
```

Comment:
If we don't change `"" if fd == 0 or flags & 0x20 else self.files[fd].name` we should at least calculate it once :P